### PR TITLE
Add more detailes to the level-6 EaaS use case

### PR DESCRIPTION
**Pull request by alexmolev**

### CODE REVIEW

File: content/trial_getting_started/examples/eaas/_index.md [hunk 1]
```diff
@@ -5,9 +5,18 @@ weight = 50
...
-determined dynamically, during runtime.
+# Level 6: Environment as a Service
+
+This example demonstrate how to use {{< param product_name >}}  [Service Component]({{< relref "working_with/service_composition/" >}}) to manage development and production environments.
+
+The [Service Component]({{< relref "working_with/service_composition/" >}}) is a node template that encapsulates a different deployment. The blueprint from which the Service Component deployment is created, as well as the new deployment's inputs, can be determined dynamically, during runtime.
+
+In this example we use a stack based on Kubernetes , Postgress as the DB and S3 as the storage system. Each of those services is referenced by a seperate service component.  The exact blueprint per component is determined at runtime based on the environment type.
+
+In a case of development environment were using a single instance per service and can also run all those services in a single VM. In the production case were using a fully managed version of those services with EKS as the Kuberentes cluster, RDS as the managed DB. and S3 as the managed storage services as illustrated in the diagram bellow.
```
Comment (ofercloudify) [inline content/trial_getting_started/examples/eaas/_index.md:16]:
```suggestion
In the case of a development environment, we are using a single instance per service and can also run all those services in a single VM. In the production case, we are using a fully managed version of those services with EKS as the Kubernetes cluster, RDS as the managed DB, and S3 as the managed storage services as illustrated in the diagram below.
```

File: content/trial_getting_started/examples/eaas/_index.md [hunk 2]
```diff
@@ -5,9 +5,18 @@ weight = 50
...
-A Service Component is a node template that encapsulates a different deployment. The blueprint
-from which the Service Component deployment is created, as well as the new deployment's inputs, can be
-determined dynamically, during runtime.
+# Level 6: Environment as a Service
+
+This example demonstrate how to use {{< param product_name >}}  [Service Component]({{< relref "working_with/service_composition/" >}}) to manage development and production environments.
+
+The [Service Component]({{< relref "working_with/service_composition/" >}}) is a node template that encapsulates a different deployment. The blueprint from which the Service Component deployment is created, as well as the new deployment's inputs, can be determined dynamically, during runtime.
+
+In this example we use a stack based on Kubernetes , Postgress as the DB and S3 as the storage system. Each of those services is referenced by a seperate service component.  The exact blueprint per component is determined at runtime based on the environment type.
```
Comment (ofercloudify) [inline content/trial_getting_started/examples/eaas/_index.md:14]:
```suggestion
In this example, we use a stack based on Kubernetes, Postgress as the DB, and S3 as the storage system. Each of those services is referenced by a separate service component.  The exact blueprint per component is determined at runtime based on the environment type.
```

**Review by ofercloudify — CHANGES_REQUESTED** [review]

**Comment by ofercloudify** [comment]
I noticed you added several png files that are not used in the article. any reason for that?

**Comment by natishalom** [comment]
> I noticed you added several png files that are not used in the article. any reason for that?

There should be a single image not multiple images

**Comment by natishalom** [comment]
@ofercloudify see the staging version - it shows a single image https://docs.cloudify.co/staging/Getting-Started-Level-6-update/trial_getting_started/examples/eaas/

**Comment by natishalom** [comment]
@ofercloudify fixed the additional files with @alexmolev help ..

**Review by ofercloudify — APPROVED** [review]

**Pull request merged**